src/components/report/ReportZipDownload.svelte

```diff
@@ -106,21 +97,121 @@
       position: absolute;
       width: 1px;
       white-space: nowrap;
-      &.focusable {
-          &:active,
-          &:focus {
-              clip: auto;
-              clip-path: none;
-              height: auto;
-              margin: 0;
-              overflow: visible;
-              position: static;
-              width: auto;
-              white-space: inherit;
-          }
+    }
+    .visuallyhidden.focusable:active,
```

Comment:
This looks like it should be pushed upstream.

Reply:
Potentially but if we allow custom CSS then we can leave the basic boilerplate and allow customization on top.